### ISSUE TITLE
Fix the hybrid/consent flows in ob-sandbox-v1

### ISSUE THREAD

**Issue by BohoCode** [SecureApiGateway/SecureApiGateway#933]
We have an issue with the hybrid/consent flows in this release! 

![image](https://user-images.githubusercontent.com/1516395/230140412-2dc3e882-e2eb-44fe-b4e3-3f3bb98bbd41.png)

[downloaded-logs-20230405-171228.zip](https://github.com/SecureApiGateway/SecureApiGateway/files/11160873/downloaded-logs-20230405-171228.zip)

_Originally posted by @BohoCode in https://github.com/SecureApiGateway/SecureApiGateway/issues/927#issuecomment-1497769687_

**Comment by BohoCode** [comment]
Failing IG forward is to http://remote-consent-service-user-interface:8080/

Looking in the cluster, the service still has the subdomain attached; 

``` 
$ k get service
NAME                                           TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)     AGE
ig                                             ClusterIP   10.30.0.76    <none>        80/TCP      65m
ob-sandbox-v1-mongodb                          ClusterIP   10.30.0.14    <none>        27017/TCP   71m
remote-consent-service                         ClusterIP   10.30.0.112   <none>        8080/TCP    65m
remote-consent-service-user-interface-rcs-ui   ClusterIP   10.30.0.254   <none>        8080/TCP    65m
test-facility-bank                             ClusterIP   10.30.0.190   <none>        8080/TCP    65m
```

**Comment by BohoCode** [comment]
Re-created the issue in my cluster.

**Comment by BohoCode** [comment]
This now works in devenv dev, so we need to fix the v1.0.0 deployment of Secure API Gateway.

To do this we will;
- Delete v1.0.0 helm chart of remote-consent-service-user-interface and create a v1.0.1 release
- Delete v1.0.0 of Secure API Gateway helm chart and modify that chart to use v1.0.1 of the remote-consent-service-user-interface (see issue #936)

**Comment by BohoCode** [comment]
Release v1.0.1 of remote-consent-service-user-interface

So contain a fix for this issue
